CMakeLists.txt

```
@@ -129,7 +129,10 @@ install(DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/src/assets/deepin-calculator
                 DESTINATION /usr/share/deepin-manual/manual-assets/application/)
 
 #if (NOT ${CMAKE_SYSTEM_PROCESSOR} MATCHES "loongarch64")
-add_subdirectory(tests)  #tests为ut测试文件夹名
+option(USE_TEST "UT" OFF)
```

CMake 的 CTest 模块提供了一个约定好的 `BUILD_TESTING` 变量可供使用，`include(CTest)` 就可以用，不需要额外定义非约定的选项名，可考虑用 `BUILD_TESTING`。

---

```suggestion
option(BUILD_TESTING "UT" ON)
```

注：不要直接在线应用修改建议，如果要在线应用的话也要确保符合 commit 信息规范，否则 commitlint 可能会失败。